ui/src/layout-editor/main.js

```diff
@@ -3609,6 +3609,13 @@ lD.openContextMenu = function(obj, position = {x: 0, y: 0}) {
     layoutObject.playlists.widgets.length > 0
   );
 
+  // Check if it's a dynamic playlist
+  layoutObject.isDynamicPlaylist =
+    (
+      layoutObject.isPlaylist &&
+      $(obj).find('.dynamic-playlist-icon').length === 1
```

Comment:
Would it be better to add something more "permanent" to this DOM element? Perhaps a data attribute or similar. I worry that if we change the icon for some reason we create a hard-to-find bug?

Reply:
Yeah, you're right. I will add a class to the main object and also for the convert playlist button on top. Thanks!